### PR TITLE
CI: bump protoc version 3.5.1 -> 3.7.1 and add native ppc64le/aarch64

### PR DESCRIPTION
**Pull request by grooverdan**
skydive-create-docker-image

### CODE REVIEW

**Comment by grooverdan** [comment]
run skydive-create-docker-image

**Comment by grooverdan** [comment]
run skydive-create-docker-image

File: contrib/docker/Dockerfile.crosscompile [hunk 1]
```diff
@@ -27,11 +27,16 @@ RUN apt-get -y update \
                    gcc-${TARGET_ARCH}-linux-gnu \
     && rm -rf /var/lib/apt/lists/*
 
-RUN PROTOC_VER=3.5.1; \
-    wget --no-verbose https://github.com/google/protobuf/releases/download/v${PROTOC_VER}/protoc-${PROTOC_VER}-linux-$(uname -m).zip \
-    && unzip protoc-${PROTOC_VER}-linux-$(uname -m).zip \
+# need to convert arch ppc64le -> ppcle_64 and aarch64 -> aarch_64
```
Comment (nplanel) [inline contrib/docker/Dockerfile.crosscompile]: please remove comment

File: contrib/docker/Dockerfile.compile [hunk 2]
```diff
@@ -18,12 +18,17 @@ RUN apt-get -y update \
 
 # EBPF requires llvm-6.0 clang-6.0 however the cross compulation docker image can't install them
 
-RUN PROTOC_VER=3.5.1; \
-    wget --no-verbose https://github.com/google/protobuf/releases/download/v${PROTOC_VER}/protoc-${PROTOC_VER}-linux-$(uname -m).zip \
-    && unzip protoc-${PROTOC_VER}-linux-$(uname -m).zip \
+# need to convert arch ppc64le -> ppcle_64 and aarch64 -> aarch_64
```
Comment (nplanel) [inline contrib/docker/Dockerfile.compile]: please remove comment

File: contrib/docker/Dockerfile.crosscompile [hunk 3]
```diff
@@ -27,11 +27,16 @@ RUN apt-get -y update \
...
+
+RUN PROTOC_VER=3.7.1; \
+    ARCH=$(uname -m); \
+    if [ $ARCH != x86_64 ]; then ARCH=${ARCH%%64*}${ARCH##*64}_64 ; fi ; \
+    FILE=protoc-${PROTOC_VER}-linux-${ARCH}.zip; \
+    wget --no-verbose https://github.com/google/protobuf/releases/download/v${PROTOC_VER}/${FILE} \
+    && unzip ${FILE} \
     && mv bin/protoc /usr/bin/ \
-    && rm protoc-${PROTOC_VER}-linux-$(uname -m).zip
+    && rm "${FILE}"
```
Comment (nplanel) [inline contrib/docker/Dockerfile.crosscompile]: && mv include/google /usr/include \

File: contrib/docker/Dockerfile.compile [hunk 4]
```diff
@@ -18,12 +18,17 @@ RUN apt-get -y update \
...
+
+RUN PROTOC_VER=3.7.1; \
+    ARCH=$(uname -m); \
+    if [ $ARCH != x86_64 ]; then ARCH=${ARCH%%64*}${ARCH##*64}_64 ; fi ; \
```
Comment (nplanel) [inline contrib/docker/Dockerfile.compile]: ARCH=${ARCH//64/}_64

Comment (grooverdan) [reply contrib/docker/Dockerfile.compile]: This is a bash only construct and in ubuntu docker build the shell is sh.

**Comment by grooverdan** [comment]
run skydive-create-docker-image

**Comment by grooverdan** [comment]
run skydive-functional-tests-backend-orientdb
run skydive-create-docker-image

**Comment by grooverdan** [comment]
run skydive-create-docker-image

**Comment by grooverdan** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by grooverdan** [comment]
Thank you.

**Pull request merged**